src/main/java/com/github/vkorobkov/jfixtures/util/YmlUtil.java

```diff
@@ -1,20 +1,43 @@
 package com.github.vkorobkov.jfixtures.util;
 
+import lombok.val;
 import org.yaml.snakeyaml.Yaml;
 
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.Collections;
 import java.util.Map;
+import java.util.stream.Stream;
 
 public final class YmlUtil {
+    private static final String YML_EXT = ".yml";
+    private static final String YAML_EXT = ".yaml";
+
     private YmlUtil() {
     }
 
     @SuppressWarnings("unchecked")
     public static Map<String, Object> load(Path file) throws IOException {
         Object loaded = new Yaml().load(Files.newInputStream(file));
-        return loaded == null ? Collections.emptyMap() : (Map<String, Object>)loaded;
+        return loaded == null ? Collections.emptyMap() : (Map<String, Object>) loaded;
+    }
+
+    public static boolean hasYamlTwin(Path yamlFilePath) {
+        val fileCount = Stream.of(YML_EXT, YAML_EXT)
+                .map(ext -> cutOffExtension(yamlFilePath.toString()) + ext)
+                .filter(name -> new File(name).exists())
+                .count();
+
+        if (fileCount == 0) {
+            throw new IllegalArgumentException("Neither " + YAML_EXT + " nor " + YML_EXT + " file extension.");
+        }
+
+        return fileCount == 2;
+    }
+
+    private static String cutOffExtension(String fullName) {
+        return fullName.substring(0, fullName.lastIndexOf('.'));
```

Comment:
It seems that if file path does not have dots the method will throw a runtime exception:
![image](https://user-images.githubusercontent.com/12804703/31823158-ceffcfbe-b5b3-11e7-9eff-0604abf5a2c1.png)

This method is good candidate to be public, to be moved out of this class and to be tested well.

Reply:
Also. directory names could contain dots:
```java
assert cutOffExtension("/home/user/.m2/config") == "/home/user/.m2/config" // but not /home/user/
```

Reply:
Probably, something like this helps:
```groovy
> import java.nio.file.Paths
> 
> def path = Paths.get("/home/vlad/.m2/me.yml")
> def parent = path.parent
> def file = path.fileName
> // todo: drop the extenstion for <file>
> parent.resolve(file)
Result: /home/vlad/.m2/me.yml
```